pkg/broker/binding_operations.go

```diff
@@ -172,15 +206,17 @@ func (b Broker) Unbind(ctx context.Context, instanceID string, bindingID string,
 func (b Broker) GetBinding(ctx context.Context, instanceID string, bindingID string) (spec domain.GetBindingSpec, err error) {
 	logger := b.funcLogger().With("instance_id", instanceID, "binding_id", bindingID)
 	logger.Infow("Retrieving binding")
+	s := BindingSpec{}
+	err = b.getState(ctx, bindingID, &s)
 
-	err = apiresponses.NewFailureResponse(fmt.Errorf("unknown binding ID %s", bindingID), 404, "get-binding")
-
-	return
+	return domain.GetBindingSpec{
```

Comment:
ok, I see that the `credentials` are used here, so this will be the service catalog implementation for getting bindings?
Then maybe these credentials should be saved into Realm secrets, not values?

Reply:
Working with Realm secrets would be a different level of pain...

We might have to rethink the whole state storage yet again if storing data in Realm is a security concern.

Reply:
I see in the docs:
```
You cannot directly read the value of a Secret after defining it
```

Maybe this is the thing that we can discuss during the next weekly meeting
@jamesbroadhead FYI!

Reply:
@antonlisovenko  thanks - yeah, let's talk through it so that we all understand. 
@antonlisovenko could you add it to the meeting agenda please? 

Am I right in my understanding that for every Atlas cluster that the OSB deploys, the credentials to access that database are stored in the realm database? 

And that the Atlas API tokens to create the clusters are also stored in the realm database?